doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
 =============================
 Multi-visit Mosaic Processing
 =============================
-Place insightful text here!
\ No newline at end of file
+
+Multi-Visit Mosaic (MVM)
+    A Multi-Visit Mosaic (MVM) is a single image (product) made by combining all observations taken of the same part of the sky.
+
+Observations taken of the same part of the sky over all the years that HST has been operational can enable unique science
+due to the high-resolution of the HST cameras.  Generating useful mosaics from these observations, though, requires
+solving a number of key problems; namely,
+
+  * aligning all the images to the same coordinate system
+  * define the size on the sky of each mosaic
+  * defining what exposures should go into each mosaic
+
+MVM processing implemented as part of the Hubble Advanced Products (HAP) pipeline generates new products based on
+solutions implemented for these critical issues.
+
+
+Alignment
+==========
+Generating MVM products relies entirely on the definition of the WCS of each input image in order to define where each
+exposure will land in the output mosaic.  Errors in an input exposure's WCS will lead to misalignment of overlapping
+images resulting in smeared sources or even multiple images of the overlapping sources in the output mosaic.  Errors
```

Comment:
In the "Errors in an" sentence the term _images_ is used with two different meanings.  The first instance refers to the exposure or 2-D frame while the second usage refers to multiple manifestations of the source.  I might change the first instance to the term _exposures_ (or something you like better).

Reply:
Changed in 241a672